tl-tl/sc/white-paper.md

```diff
@@ -117,91 +117,92 @@ Ang NeoVM ay nagbibigay ng isang birtwal na hardware layer, upang sumuporta ng m
 
  * **CPU**
 
- CPU is responsible for reading and sequentially order the execution of instructions in the contract, according to the function of the instruction flow control, arithmetic operations, logic operations. The future of the CPU function can be extended, with the introduction of JIT (real-time compiler) function, thereby enhancing the efficiency instruction execution. 
+ Ang CPU ay responsable para sa pagbabasa at magkakasunod na pagkakaayos ng pagpapatupad ng mga instruksyon sa kontrata, ayon sa punsyon ng flow control ng instruksyon, aritmetikang mga operasyon, lohikang mga operasyon. Ang hinaharap ng CPU na punsyon ay maaaring mapalawak, gamit ang pagpapakilala ng JIT (real-time na kompayler) na punsyon, sa gayon ay nagpapabuti sa kahusayan na pagpapatupad ng instruksyon.
 
  * **Call Stack**
 
-   The call stack is used to hold the context information of the program execution at each function call, so that it can continue to execute in the current context after the function has finished executing and returning.
+   Ang call stack ay ginagamit upang maghawak ng mga kontekstong impormasyon ng pagpapatupad ng program sa bawat pagtawag ng punsyon, upang ito ay maaaring magpatuloy sa pagpapatupad sa kasalukuyang konteksto pagkatapos ng pagpapatupad at pagsasauli ng punsyon.
 
  * **Calculate Stack**
 
-   All NeoVM run-time data are stored in the calculation stack, when after the implementation of different instructions, the stack will be calculated on the corresponding data elements of the operation. For example, when additional instructions are executed, the two operations participating in the addition are ejected from the calculation stack, and the result of the addition is pushed to the top of the stack. Function call parameters must also be calculated from right to left, according to the order of the stack. After the function is successfully executed, the top of the stack fetch-function returns the value.
+   Ang lahat ng run-time na datos ng NeoVM ay naka-imbak sa kalkulasyon na stack, kapag pagkatapos ng implementasyon ng magkaibang mga instruksyon, ang stack ay ikakalkula sa nararapat na mga elemento ng datos sa operasyon. Halimbawa, kapag ang karagdagang mga instruksyon ay naipatupad, ang dalawang mga operasyon na lumalahok sa pagdaragdag ay papaalisin mula sa kalkulasyon na stack, at ang resulta ng pagdaragdag ay itutulak sa itaas ng stack. Ang mga parameter sa pagtawag ng punsyon ay dapat ding makalkula mula sa kanan patungo sa kaliwa, ayon sa pakakaayos ng stack. Pagkatapos ng matagumpay na pagpatupad ng punsyon, ang nasa itaas ng stack fetch-function ay magsasauli ng halaga.
```

Comment:
`kapag pagkatapos ng` to `kung kailan matatapos ang`

Reply:
`elemento ng datos` to `elemento na datos`

Reply:
`parameter` to `parametro`

Reply:
done